Dockerfile

```
@@ -1,11 +1,11 @@
-FROM golang:1.13.4 AS build
+FROM golang:1.16 AS build
 WORKDIR /src
 ENV GO111MODULE=on
 RUN go get -u github.com/swaggo/swag/cmd/swag@v1.6.5
 COPY . /src
 RUN swag init && CGO_ENABLED=0 go build -o main
 
-FROM alpine:3.13.4 AS final
+FROM alpine:3.13 AS final
```

We always want to use as percise version as possible as we've been bitten too many times from version ranges and `latest` tags. Suggest changing this to `alpine:3.13.5` instead

```suggestion
FROM alpine:3.13.5 AS final
```